api/src/organizational-entities/infrastructure/repositories/certification-center-for-admin.repository.js

```diff
@@ -22,7 +23,28 @@ const update = async function (certificationCenter) {
     .where({ id: certificationCenter.id });
 };
 
-export { save, update };
+/**
+ * @type {function}
+ * @param {Object} params
+ * @param {string|number} params.certificationCenterId
+ * @param {string|number} params.archivedBy
+ * @param {date} params.archiveDate
+ */
+const archive = async function ({ certificationCenterId, archivedBy, archiveDate }) {
+  const knexConn = DomainTransaction.getConnection();
+  const certificationCenter = await knexConn('certification-centers').where({ id: certificationCenterId }).first();
+  if (!certificationCenter) {
+    throw new CertificationCenterNotFoundError();
```

Comment:
Utiliser l'exception générique `NotFoundError` et ne pas définir une nouvelle `CertificationCenterNotFoundError` pour laquelle il n'y a pas de besoin avéré : 

```suggestion
    throw new NotFoundError();
```